packages/google_workspace/data_stream/admin/manifest.yml

```
@@ -0,0 +1,27 @@
+type: logs
+title: Admin logs
+release: experimental
+streams:
+  - input: httpjson
+    template_path: httpjson.yml.hbs
+    title: Admin logs (httpjson)
+    description: Collect admin logs using httpjson input
+  - input: logfile
```

If this one is used only for testing I suggest to add:
```suggestion
  - input: logfile
    enabled: false
```